service/lib/ops/policyOps.js

```diff
@@ -39,16 +39,26 @@ function insertPolicies (client, policies, cb) {
   client.query(sql, cb)
 }
 
-function deletePolicies (client, ids, cb) {
-  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deletePolicies (client, ids, orgId, cb) {
+  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids}) AND org_id=${orgId}`, utils.boomErrorWrapper(cb))
```

Comment:
Do you know if who is calling the `deletePolicies` check if we actually removed anything? (ie: https://github.com/nearform/labs-authorization/blob/37affde7691cd3e1ad56aa5c365c2218546f767b/service/lib/ops/userOps.js#L416 )